projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-child-rows/table-child-rows.component.html

```diff
@@ -0,0 +1,95 @@
+<section class="go-container">
+
+  <go-card class="go-column go-column--100" id="implementing-child-rows">
+    <ng-container go-card-header>
+      <h2 class="go-heading-2 go-heading--no-wrap">Implementing Child Rows</h2>
+      <go-copy cardId="implementing-child-rows" goCopyCardLink></go-copy>
+    </ng-container>
+    <ng-container go-card-content>
+      <p class="go-body-copy">
+        Sometimes within a table row we might need to show child rows related to the row. This is where the table child rows comes
+        into play. This provides a dropdown for each table row that when opened will render child rows as defined by the implementor.
+        With the table child rows template we allow the implementor to provide a template for each column to use for the row and then we inject
+        that template into the correct place in the interface, this works exactly like the normal table columns.
```

Comment:
Just a minor grammatical note: should "this" be the start of a new sentence? Or perhaps it could be put in parentheses:

"With the table child rows template we allow the implementor to provide a template for each column to use for the row and then we inject that template into the correct place in the interface (this works exactly like the normal table columns)."

Reply:
Fair enough, I cleaned up the grammar here a bit.